client/images1.jsx

```diff
@@ -1,12 +1,41 @@
 import React from 'react';
+import styles from './styles.css.js';
 
 const images1 = (props) => (
 	<div>
-		{Object.keys(props.amenities).map((amenity) => {
-			<div>
-				<img name={amenity} src={props.images[amenity]}></img>
-				<label class="images1" for={amenity}>{amenity}</label>
-			</div>})}
+	<h1>Special Amenities</h1>
+		{Object.keys(props.special).map((amenity) => (
+			props.special[amenity] ?
+			<div style={styles.special}>

```

Comment:
Maybe write your ternary in the format for better reading.
condition
? do this
: do this